.github/workflows/android.yml

```diff
@@ -52,7 +50,7 @@ jobs:
       - name: Extract symbols
         run: |
           DUMP_SYMS=./core/deps/breakpad/bin/dump_syms
-          SO_DIR=shell/android-studio/flycast/build/intermediates/cmake/release/obj
+          SO_DIR=shell/android-studio/flycast/build/intermediates/merged_native_libs/release/out/lib
```

Comment:
- The path `shell/android-studio/flycast/build/intermediates/cmake/release/obj` does not exist anymore with the update.
- Sentry usesthe path `build/intermediates/merged_native_libs/{buildVariant}` in their documentation https://docs.sentry.io/platforms/android/data-management/debug-files/upload/